linera-execution/src/lib.rs

```diff
@@ -1028,15 +1028,14 @@ impl<Message> RawOutgoingMessage<Message, Resources> {
 }
 
 impl OperationContext {
-    fn refund_grant_to(&self) -> Option<Account> {
-        // TODO: return `Account` instead.
-        Some(Account {
+    fn refund_grant_to(&self) -> Account {
```

Comment:
TODO: double-check. I think this is wrong b/c previously `None` was kept as `None` and rn we use `AccountOwner::Chain`.

Reply:
This is correct. It always returned `Some(_)` and `owner: None` was already refunding chain.

Reply:
The option type controls whether we refund or not.

Reply:
Yes, and that part is kept intact. Whereas this method used to always return `Some(_)` I know wrap the returned value with `Some(_)`.